util/cron/test-gasnet.llvm.bash

```diff
@@ -5,7 +5,7 @@
 
 CWD=$(cd $(dirname $0) ; pwd)
 source $CWD/common-gasnet.bash
-source $CWD/common-llvm.bash
+source $CWD/common-llvm.bash llvm
```

Comment:
I have a slight preference for not modifying scripts that are testing `CHPL_LLVM=llvm`, but I won't push for that if you think it's worthwhile for some reason. 

If you do want to modify them, please update `util/cron/test-linux64-vm.bash` too

Reply:
The reason I changed the other llvm scripts was just to help remind us (next time we copy-paste one of these test-llvm.bash things) that common-llvm can now set up for LLVM=llvm or LLVM=system via the new commandline parameter. This was not a strong reason, so we'll just forget it.